PxAPI-2.yml

```diff
@@ -802,6 +806,257 @@ components:
+              type: string
+        tags:
+          type: array
+          items: 
+            type: string
+        discontinued:
+          type: boolean
+        contact:
+          type: array
+          description: A list of contacts associated with the table.
+          items: 
+            $ref: '#/components/schemas/Contact'
+    extension:
+      description: extension at some level
+      type: object
+    extension-dimension:
+      description: extension at dimension
+      type: object
+      properties:
+        elimination:
+          type: boolean
+        eliminationValueCode:
+          type: string
+        note:
+          type: array
+          items:
+            $ref: '#/components/schemas/Note'
+        valueNote:
+          type: object
+          additionalProperties:
+            type: array
+            items:
+              $ref: '#/components/schemas/Note'          
+        filters:
+         $ref: '#/components/schemas/filters'
+        frequency:
+          type: string
+          description: Hmm, we have this on root as well
+        firstPeriod: 
+          type: string
+        lastPeriod: 
+          type: string
+    jsonstat-note:
+      "$ref": "#/components/schemas/strarray"
+    strarray-by-str-dict:
+       type: object
+       additionalProperties:
+          type: array
+          items:
+            type: string
+          uniqueItems: true  
+    jsonstat-link:
+      type: object
+      additionalProperties: 
+         type: array
+         items:
+            type: object
+            title: jsonstatLink
+            properties:
+              type:
+                type: string  
+              href:
+                "$ref": "#/components/schemas/href"
+            additionalProperties: false
+    jsonstat-extension-link:
+      type: object
+      properties:
+        describedby:
+          type: array
+          items:
+            type: object
+            title: DimensionExtension
+            properties:
+              extension:
+                 description: A extension object
+                 type: object
+                 additionalProperties:
+                    description: A string
+                    type: string
+                    
+            additionalProperties: false       
+      additionalProperties: false         
+
+    jsonstat-category:
+      type: object
+      properties:
+        index:
+           type: object
+           additionalProperties:
+                type: integer
+        label:
+          type: object
+          minProperties: 1
+          additionalProperties:
+            type: string
+        child:
+           "$ref": "#/components/schemas/strarray-by-str-dict"
+        unit:
+          type: object
+          additionalProperties:
+            type: object
+            properties:
+              base:
+                type: string
+              decimals: 
+                type: integer
+      additionalProperties: false
+    filters:

```

Comment:
In the custom JSON we changed filters to codelists and do not distinct between vs and agg do we whant to do the same for JSON-STAT2?

Reply:
Thanks, I think the jsonstat example used was made when codelists were filters. There are some other changes  to Dataset coming ....

Reply:
Changed filters to codelists. Moved note for table to extension to support mandatory notes. refperiod  moved to extension for dimension.

Ready for review :)